includes/feature-stage-beta-user-feedback.mdx

```diff
@@ -1,5 +0,0 @@
-<Note title="A Wild User Feedback Appears!">
-
-Sentry released a new version of User Feedback that supports collecting feedback at any time from your users, without requiring an error event. It is currently in open beta and subject to change.
```

Comment:
i wonder if we can re-utilize this to say UF is in GA now

Reply:
oh a banner for that could be nice! Maybe add in a new file + PR though?

Reply:
We could :o Not something we've done in the past for docs though, so I'd make sure to get the docs teams eyes on it for their opinion